modules/ipi-install-provisioning-the-bare-metal-node.adoc

```diff
@@ -10,47 +10,47 @@ Provisioning the bare metal node requires executing the following procedure from
 
 .Procedure
 
-. Ensure the `STATE` is `ready` before provisioning the bare metal node.
+
+. Ensure the `PROVISIONING STATUS` is `Available` before provisioning the bare metal node.
 +
-[source,bash]
+[source,terminal]
 ----
 $  oc -n openshift-machine-api get bmh openshift-worker-<num>
 ----
 +
 Where `<num>` is the worker node number.
 +
-[source,bash]
+[source,terminal]
 ----
-NAME                     STATE   CONSUMER     ONLINE   ERROR
-openshift-worker-<num>   ready                true
+NAME              STATE     CONSUMER                                ONLINE ERROR  AGE
+openshift-worker  available ocp-edge-cluster-0-8rvrp-worker-0-g92k4 true          34h
```

Comment:
```suggestion
openshift-worker  available           true          34h
```
when bmh in available state, there should be no consumer name since no machine consumes the host